modules/ztp-configuring-kernel-arguments-for-discovery-iso-in-manual-installations.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-deploying-far-edge-sites.adoc
+
+[id="setting-managed-bare-metal-host-kernel-arguments_{context}"]
+= Configuring kernel arguments for the Discovery ISO for manual installations by using GitOps ZTP
```

Comment:
This heading seems a bit convoluted to me. How about:
_Configuring kernel arguments for the Discovery ISO by using GitOps ZTP for manual installations_ 

That way you have a _Configuring kernel arguments for the Discovery ISO by using GitOps ZTP_ for the RHACM method and one for manual install. Just a thought.

Reply:
What about:

```suggestion
= Configuring Discovery ISO kernel arguments for manual installations using GitOps ZTP
```